lib/liquider/erb_compiler.rb

```diff
@@ -124,6 +124,18 @@ def on_for(for_node)
     variable_name = "_liquider_var_#{@variable_number += 1}"
     erb_tag {
       for_node.expression.visit(self)
+      unless for_node.offset.is_a?(NumberNode) && for_node.offset.value == 0
```

Comment:
So the order is `offset` -> `reversed` -> `limit`? Do you think it is worth submitting PRs to the liquid doc when we stumble on corner cases like this?